src/rgw/rgw_op.cc

```diff
@@ -1045,13 +1045,10 @@ int handle_cloudtier_obj(req_state* s, const DoutPrefixProvider *dpp, rgw::sal::
         ldpp_dout(dpp, 5) << "restore is not completed yet, please retry again" << dendl;
```

Comment:
Can we change this message to "restore is initiated, please check restore status and retry"?